pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>mod-patron</artifactId>
-  <version>4.4.0-SNAPSHOT</version>
+  <version>4.4.1-SNAPSHOT</version>
```

Comment:
Why has the version been changed?

Reply:
Hm, I thought it's need to increase version of snapshot too, seems I saw this somewhere in other modules too. But it seems this doesn't make sense because that's identified of snapshot version.  Thank you Marc for pointing this.